Dockerfile

```diff
@@ -443,7 +443,7 @@ RUN \
 
 FROM sdk-libc as sdk-go
 
-ENV GOVER="1.21.5"
+ENV GOVER="1.21.7"
```

Comment:
They just released a newer version with security fixes:

https://github.com/golang/go/issues?q=milestone%3AGo1.21.8+label%3ACherryPickApproved

Reply:
Correct. I went ahead and opened/merged https://github.com/bottlerocket-os/bottlerocket-sdk/pull/158 so this could rebase on it.